src/main/archetype/ui.config/src/main/content/META-INF/vault/filter.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<workspaceFilter version="1.0">
+    <filter root="/apps/${appId}/config"/>
```

Comment:
This gets overwritten by https://github.com/adobe/aem-project-archetype/blob/master/src/main/archetype/ui.apps/src/main/content/META-INF/vault/filter.xml. So you need to add an exclude to `ui.apps` and for simplicity reasons I would put al configs one level deeper, i.e. `/apps/${appId}/osgiconfig/config.author` and `/apps/${appId}/osgiconfig/config.prod`, ...